storage/time-series/influxdb/storer/app/influx/index.js

```diff
@@ -73,32 +75,49 @@ class InfluxDB {
   }
 
   /**
- *  Returns a Measurement instance
- * @returns {Measurement}
- */
+   *  Returns a Measurement instance
+   * @returns {Measurement}
+   */
   getInfluxMeasurementInstance() {
     return this.influxMeasurement;
   }
 
   /**
-     *  Returns a State instance
-     * @returns {State}
-     */
+   *  Returns a State instance
+   * @returns {State}
+   */
   getInfluxStateInstance() {
     return this.influxState;
   }
 
   /**
    * Creates a 'healthCheck' for influxDB
+   *
+   * @param {function()} cbHealth Callback to be called if the service is
+   *                              healthy, this.isHealth is not null
+   *                              and if the value of this.isHealth is to change.
```

Comment:
I don't understand this `if the value of this.isHealth is to change.`

Reply:
If the value of the variable is to change in that call, that is, the state of the application is to change from healthy to unhealthy in that call for example.